src/Recurring.php

```diff
@@ -104,7 +138,24 @@ public static function pause($id, $options = [])
      * @param string $id      recurring payment ID
      * @param array  $options user's headers
      *
-     * @return array
+     * @return array[
+     * 'id'=> string,
+     * 'user_id'=> string,
+     * 'external_id'=> string,
+     * 'status'=> string,
```

Comment:
Is it possible to create the `status` properties return something like enum? So the user can expect the strings that will be returned. If it's not possible, then it's fine

Reply:
I don't think it's possible, but it can be like `'status' => 'ACTIVE' || 'STOPPED' || 'PAUSED'`

Reply:
Nah that one. I think it is better. So the user doesn't need to do if/else to check (let's assume the user are really careful haha)